ui/components/multichain/account-picker/account-picker.js

```diff
@@ -37,74 +37,68 @@ export const AccountPicker = ({
   addressProps = {},
   labelProps = {},
   textProps = {},
+  className = '',
```

Comment:
TODO: Add unit test to cover this prop

Reply:
Done here https://github.com/MetaMask/metamask-extension/pull/21544/commits/95a974ce82afce3465209fd4a8eafae2b4669fc5